templates/sag-db-sqlserver/README.md

```diff
@@ -31,23 +30,21 @@ sagcc exec templates composite apply sag-db-sqlserver
 
 ## Using for local development and testing on Docker platforms
 
-To use this template for local development launch Command Central container first.
-From the root of this project run:
+To use this template for local development you must first launch the Command Central container. From the root folder of this project run:
 
 ```bash
 docker-compose up -d cc
 ```
 
-Run this command to launch [Miscrosoft SQL Server on Linux for Docker Engine](https://hub.docker.com/r/microsoft/mssql-server-linux/)
-container and apply the template to create the user, database and IS, MWS, and BPM schemas for 10.3 release:
+To launch the [Miscrosoft SQL Server on Linux for Docker Engine](https://hub.docker.com/r/microsoft/mssql-server-linux/) container and apply the `sag-db-sqlserver` template to create a user, database, and IS, MWS, and BPM database schemas for the 10.3 release:
```

Comment:
To launch the [Miscrosoft SQL Server on Linux for Docker Engine](https://hub.docker.com/r/microsoft/mssql-server-linux/) container and apply the `sag-db-sqlserver` template to create a user, database, and Integration Server and My webMethods Server database components for the 10.3 release: